libc/src/setjmp/x86_64/setjmp.h

```diff
@@ -0,0 +1,22 @@
+
+// TODO: license block
+#ifndef LLVM_LIBC_SRC_SETJMP_X86_64_SETJMP_H
+#define LLVM_LIBC_SRC_SETJMP_X86_64_SETJMP_H
+
+#ifdef __ASSEMBLER__
+#define UL(x) x
+#else
+#define UL(x) x##UL
+#endif
```

Comment:
Will be useful again, move to assembly.h

Reply:
hmm...maybe not.  I do have a:

```
   11 #ifndef __ASSEMBLER__                                                                                                                                                                                                               
  12 #error "No not include assembly.h in non-asm sources"     ■ "No not include assembly.h in non-asm sources"                                                                                                                          
   13 #endif 
```
in assembly.h. Maybe I should change that so that it can be included in either sources. Or create yet another new header.